src/invoker.ts

```diff
@@ -258,8 +258,8 @@ export class ErrorHandler {
       logAndSendError(err, latestRes, killInstance);
     });
 
-    process.on('exit', code => {
```

Comment:
I think this would remove error logging when the process exits from user's function.

I do think we should not log and send error when process exits normally though. How about checking the status code and only `logAndSendError` for non-0 ones? Or handing `SIGINT` before `exit`?